src/components/Explorer.vue

```diff
@@ -749,15 +762,79 @@ export default {
     await this.determineFilterCountsAndSubmit();
     await this.fetchPage();
   },
+  async postData(url = "", data = {}) {
+    // Default options are marked with *
+    const response = await fetch(url, {
+      method: "POST", // *GET, POST, PUT, DELETE, etc.
+       headers: {
+        "Content-Type": "application/json",
+        // 'Content-Type': 'application/x-www-form-urlencoded',
+       },
+       body: JSON.stringify(data), // body data type must match "Content-Type" header
+    });
+    return response.json(); // parses JSON response into native JavaScript objects
+  },
   // Browser only
   async mounted() {
+    if ("geolocation" in navigator) {
+      navigator.geolocation.getCurrentPosition(async (position) => {
+        console.log("POSITION", position)
```

Comment:
Probably should remove this log now that you're done debugging?

Reply:
Suggestion taken